blueprints/service-test/index.js

```diff
@@ -1,9 +1,37 @@
 'use strict';
 
 const useTestFrameworkDetector = require('../test-framework-detector');
+const { isModuleUnificationProject } = require('../module-unification');
 
 module.exports = useTestFrameworkDetector({
   description: 'Generates a service unit test.',
+
+  fileMapTokens: function() {
```

Comment:
You can use shorthand methods I think, since only node 4 is supported (and in 3 days, only node 6! 🎉 🎉 🎉 )

Reply:
Sadly I think its actually the end of April 😭

Reply:
crap, i thought it was the 1st 😭

Reply:
anyway shorthand methods are [supported](https://node.green/#ES2015-syntax-object-literal-extensions-shorthand-methods) in node 4 😄

Reply:
anyway node 4 [supports](https://node.green/#ES2015-syntax-object-literal-extensions-shorthand-methods) shorthand methods 😄